test/functional/wallet_groups.py

```diff
@@ -25,9 +25,10 @@ def set_test_params(self):
             ["-maxapsfee=0.00000293"],
             ["-maxapsfee=0.00000294"],
         ]
-        # whitelist peers to speed up tx relay / mempool sync
+
```

Comment:
25990: CI fails for wallet_groups.py (all jobs that have wallet functional tests)

```
wallet_groups.py --legacy-wallet                   | ✖ Failed  | 20 s
wallet_groups.py --descriptors                     | ✖ Failed  | 14 s
```

https://gitlab.com/dashpay/dash/-/jobs/9856673673